.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+on:
+    pull_request:
+  
+jobs:
+    version_matrix:
+      strategy:
+        matrix:
+          os: [ubuntu-latest, windows-latest]
+          python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Python 3.7 is past end-of-life and should be removed.

Reply:
Good catch!